Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -249,7 +259,7 @@ private void UpdateRotation()
     {
         aimAngle += inputManager.lookInput * lookSpeed * Time.deltaTime;
         // Constrain aiming angle vertically and wrap horizontally.
-        aimAngle.y = Mathf.Clamp(aimAngle.y, -Mathf.PI / 2, Mathf.PI / 2);
+        aimAngle.y = Mathf.Clamp(aimAngle.y, -Mathf.PI / 2 + Mathf.Deg2Rad, Mathf.PI / 2 - Mathf.Deg2Rad);
```

Comment:
Why's Deg2Rad here now, and as an additive term??? It seems to work though, but isn't that just because 2Π/360 is a very small value?

Reply:
If we lock it at the very top hand animations tend to break due to shortest path slerping. This is a very unnoticable (in game) and computationaly inexpensive way to solve this issue

Reply:
Okay, but why not set it to some custom low value constant instead of using Deg2Rad, which I suspect would confuse others just as much about the intent of this slight alteration

Reply:
My intention is to move it 1 degree, so I could go 
`
ROTATION_OFFSET = 1;

aimAngle.y = Mathf.Clamp(aimAngle.y, -Mathf.PI / 2 + Mathf.Deg2Rad * ROTATION_OFFSET, Mathf.PI / 2 - Mathf.Deg2Rad * ROTATION_OFFSET);
`
But that seems like very overkill considering Mathf.Deg2Rad is literally a constant value of 1 degree in radians, and we probably dont what to changer this value very often

Reply:
I can write a dev comment  above the function though, if you think the readability is suffering too much atm

Reply:
Aha, that sounds quite sensible actually, but I do think there should be a comment explaining what the intent is (as in, that Deg2Rad is used for offsetting by 1 degree and why offsetting is necessary)